js/script.js

```diff
@@ -0,0 +1,15 @@
+const navSlide = () => {
+  const burger = document.querySelector('.humberger-menu');
+  const nav = document.querySelector('.desktop-nav');
+  burger.addEventListener('click', () => {
+    nav.classList.toggle('nav-active');
+    burger.classList.toggle('toggle');
+  });
+
+  document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+    nav.classList.remove('nav-active');
+    burger.classList.remove('toggle');
+  }));
+};
+
+navSlide();
```

Comment:
- [ ] When I click on the mobile menu options, the drop-down menu doesn't disappear. Please make sure when the user clicks any of the mobile menu options, the mobile menu disappears and the correct part of the page is displayed.

![image](https://user-images.githubusercontent.com/83803414/195100542-b8319c50-97fe-4297-8d7f-f6a01fbb5eac.png)